app/assets/javascripts/rad_common/dynamic_updater.js

```diff
@@ -0,0 +1,14 @@
+$(document).ready( function() {
+    $('form.dynamic-updater input').change(function() {
+        let url = $(this).closest('form').attr('action') + ".json";
+        let data = $(this).closest('form').serializeArray();
+        $.post(url, data)
+            .fail(function(data) {
+                addAlert('danger', data.responseJSON.error)
+            });
+    });
+
+    function addAlert(type, message) {
+        $('.card-body').prepend( $("<div class='alert alert-" + type + "'>" + message + "</div>") )
+    }
+});
```

Comment:
Not sure if you would rather coffee script here but I have kind of been preferring using plain JS instead as coffee script tends to take me more time to write and understand, but I can convert it to coffee if you prefer.